.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseDurationParser.cs

```diff
@@ -228,7 +228,7 @@ private DateTimeResolutionResult ParseAnUnit(string text)
                     var unitStr = this.config.UnitMap[srcUnit];
 
                     ret.Timex = TimexUtility.GenerateDurationTimex(numVal, unitStr, IsLessThanDay(unitStr));
-                    ret.FutureValue = ret.PastValue = double.Parse(numStr) * this.config.UnitValueMap[srcUnit];
+                    ret.FutureValue = ret.PastValue = double.Parse(numStr, CultureInfo.InvariantCulture) * this.config.UnitValueMap[srcUnit];
```

Comment:
The problem with this was that it return wrong values for e.g. "half year". In this class the numStr is created with 

 It would create parse this to 0,5 year

Reply:
You're actually correct. This can change should stay. My comment applies only to Unit.